src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -156,7 +156,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                             }
                             else
                             {
-                                _logger.LogWarning(0, ex, $"Unable to bind to {parsedAddress.ToString()} on the IPv4 loopback interface.");
+                                _logger.LogInformation(0, $"Unable to bind to {parsedAddress.ToString()} on the IPv4 loopback interface.");
```

Comment:
Because if you have full control over the host, you could see why binding to either IPv4 or IPv6 failed and then fix it. Typically it's some OS-level configuration, no?

You could also change "localhost" to "127.0.01" or "[::1]" in `UseUrls()` if you expect to only bind to one interface.